crawl-ref/source/religion.cc

```diff
@@ -340,13 +340,15 @@ const vector<god_power> god_powers[NUM_GODS] =
 
     // Wu Jian
     { { 1, "attack monsters by moving around them",
-           "no longer perform spinning attacks" },
+           "perform spinning attacks" },
```

Comment:
Why not just remove these extra entries entirely, as, for example, Hepliaklqana does?

Reply:
"You can no longer perform airborne attacks." sounds better than "You can no longer perform distracting airborne attacks by moving against a solid obstacle."
This holds true for all other abilities except maybe the first one.